common/components/componentsBySection/FindProjects/ProjectCardsContainer.jsx

```diff
@@ -52,6 +60,35 @@ class ProjectCardsContainer extends React.Component<{||}, State> {
             />
         );
   }
+
+  _handleFetchNextPage(e: object): void {
+    e.preventDefault();
+    this.setState({current_page: this.state.current_page + 1 <= this.state.project_pages 
+      ? this.state.current_page + 1 
+      : this.state.current_page }, function () {
+      ProjectSearchDispatcher.dispatch({
+        type: 'SET_PAGE',
+        page: this.state.current_page,
+      });
+    });
+  }
+
+  _renderPagination(): React$Node {
```

Comment:
```suggestion
  _renderPagination(): ?React$Node {
```
To signal that we might return null.